### PR TITLE
changes in merge_sorted_list added

### PR DESCRIPTION
**Pull request by NinaPatrina**

### CODE REVIEW

**Review by emilysaeli — APPROVED** [review]
Did you copy this code?

File: tests/merge_sorted_lists_test.py [hunk 1]
```diff
@@ -1,2 +1,38 @@
 import pytest
 from functions.merge_sorted_lists import merge_sorted_lists
```
Comment (emilysaeli) [inline tests/merge_sorted_lists_test.py:2]:
```suggestion
from functions.merge_sorted_lists import merge_sorted_lists
```

Comment (NinaPatrina) [reply tests/merge_sorted_lists_test.py:2]: Do not see any difference sorry

**Comment by NinaPatrina** [comment]
> 

yES

**Pull request merged**